### ISSUE TITLE
How do I open it?

### ISSUE THREAD

**Issue by Shawnyboy992** [MichaelBest01/michaelbest01.github.io#4]
Sorry im a lil slow but how do i get to the website?

**Comment by MichaelBest01** [comment]
You need to press a circle on the bottom left or you need to do a ctrl m as
the shorcut. I'm not sure about the shortcut since I haven't used it in a
while.

**Comment by Shawnyboy992** [comment]
Ik know but like i need to get to the thing to do that. I know how to get past the cloak but the problem is i cant get to the cloak in order to do allat. Sorry for the misunderstanding

**Comment by Shawnyboy992** [comment]
And thanks for responding so quickly!

**Comment by MichaelBest01** [comment]
If you got past the cloak and see the site then you are fine

**Comment by Shawnyboy992** [comment]
Yes but I cant get into the cloak, Im still on the looking for it in github part lol. Can you guide me through?

**Comment by MichaelBest01** [comment]
Okay go to the site. michaelbest01.github.io and you will see that the site
cant be reached. That's the cloak. Press on the bottom left corner or press
ctrl m

**Comment by Shawnyboy992** [comment]
Ok, Thank you so much!
Sorry for the trouble!
